tools/install-powershell-readme.md

```diff
@@ -66,3 +66,29 @@ bash <(wget -O - https://raw.githubusercontent.com/PowerShell/PowerShell/master/
 ### Installation To do list
 
 * Detect and wait when package manager is busy/locked? - at least Ubuntu (CentOS does this internally)
+
+# install-powershell.ps1
+
+## Features of install-powershell.ps1
+
+* can be called directly from git
+* optionally allows install of the latest Preview build
+* optionally allows install of the Daily build
+* optionally installs using the latest MSI 
+* automatically looks up latest version via git tags
+* optionally installs silently
+* optionally adds the install location to Path environment variable
+
+## Examples
+
+### Install PowerShell Core Daily Build
+
+```PowerShell
+iex "& { $(irm 'https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/install-powershell.ps1') } -daily"
```

Comment:
The URL can be replaced by https://aka.ms/install-powershell.ps1 which points to the same location.